tests/cases/fourslash/declarationExpressions.ts

```diff
@@ -0,0 +1,23 @@
+/// <reference path="fourslash.ts"/>
+
+////class A {}
+////const B = class C {
+////    public x;
+////};
+////function D() {}
+////const E = function F() {}
+////console.log(function inner() {})

```

Comment:
can you add another test with some nesting, class expression inside a function expression